share/spice/tides/tides.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_tides = function(api_result){
+        if (!api_result || !api_result.tide) {
```

Comment:
It looks like there could be an `error` property that you can check for

![screen shot 2015-08-29 at 4 28 27 pm](https://cloud.githubusercontent.com/assets/81969/9564162/0069d584-4e6b-11e5-8eb3-85b8ae6d8db8.png)